lib/cms/repo.ex

```diff
@@ -104,7 +105,7 @@ defmodule CMS.Repo do
               cache: @cache,
               key: "/cms/whats-happening",
```

Comment:
This key was changed to `cms.repo|whats-happening` in another PR. Same as the one below. So, the one above is really just adding a key when we used a generated one before.